test/foo/lib/config/test_configloader.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+from foo.lib.config import ConfigLoader, ConfigLoaderException
+
+
+def test_load(monkeypatch):
```

Comment:
This test case could be divided into two, one to check if config values are overwritten with environmental variables and one to check if defaults are correctly appended without modifying values in the config.